index.bs

```diff
@@ -1,11 +1,62 @@
-<pre class='metadata'>
-Title: Web Extensions
-Shortname: webextensions
-Level: 1
-Status: w3c/CG-DRAFT
-Group: WECG
-URL: https://w3c.github.io/webextensions
-Editor: Mukul Purohit, Microsoft Corporation https://www.microsoft.com, mpurohit@microsoft.com
-Editor: Tomislav Jovanovic, Mozilla https://www.mozilla.org/, tjovanovic@mozilla.com
-Abstract: [Placeholder] Abstract.
-</pre>
+<pre class='metadata'>
+Title: Web Extensions
+Shortname: webextensions
+Level: 1
+Status: w3c/CG-DRAFT
+Group: WECG
+URL: https://w3c.github.io/webextensions
+Editor: Mukul Purohit, Microsoft Corporation https://www.microsoft.com, mpurohit@microsoft.com
+Editor: Tomislav Jovanovic, Mozilla https://www.mozilla.org/, tjovanovic@mozilla.com
+Abstract: [Placeholder] Abstract.
+Markup Shorthands: markdown yes
+</pre>
+
+# Manifest
+
+A WebExtension must have a manifest file at its root directory.
+
+## Manifest file
+
+A manifest file is a [[!JSON]] document named `manifest.json`. Malformed JSON files are not supported.
+
+## Manifest keys
+
+If manifest keys that are not defined in this specification are specified, implementors may ignore those keys.
```

Comment:
"…implementors _should_ ignore those keys." since we want a developer to be able to show one manifest to any browser without worry it will be flagged as invalid in some browsers.

Reply:
I changed it to «must» without thinking too much about it, I think this should discussed for real afterwards, maybe in a separate issue as well?

Reply:
+1 to more issues :)